core/vesting/vesting.go

```diff
@@ -372,7 +372,7 @@ func (e *Engine) broadcastSummary(ctx context.Context, seq uint64) {
 					&eventspb.PartyLockedBalance{
 						Asset:      asset,
 						Balance:    balance.String(),
-						UntilEpoch: seq + remainingEpochs,
+						UntilEpoch: seq + remainingEpochs + 1, // we add one here because the remainingEpochs can be 0, meaning the funds are released next epoch
```

Comment:
does it mean that? doesn't 0 mean they are not parked in the vesting account but move in the same epoch to the vested?

Reply:
well, it depends. when calling AddReward: https://github.com/vegaprotocol/vega/blob/develop/core/vesting/vesting.go#L151 then yes.

But in here: https://github.com/vegaprotocol/vega/blob/develop/core/vesting/vesting.go#L238-L246
We can have some with 0, which means it get distributed next epoch.

Reply:
in any case if it's 0 when sent to AddRewad, it never get locked, and go through this process.

Reply:
ok cool, so it's just the internal accounting of the vesting engine

Reply:
yes.